composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "zf1s/zf1",
+    "name": "yapla/zf1",
```

Comment:
ok. Obviously I don't want to merge this here but I needed this definition for `composer require` my forked version :think: